app/src/main/res/values-zh-rTW/strings.xml

```diff
@@ -602,4 +602,5 @@
   <string name="archive_password_prompt">請輸入壓縮檔的密碼。</string>
   <string name="cannot_extract_archive">不能開啟壓縮檔 \"%s\"： %s</string>
   <string name="archive_unsupported_or_corrupt">不能開啟壓縮檔 \"%s\"。可能是程式庫不支援此壓縮檔，或者壓縮檔真的壞掉了。</string>
+  <string name="close">關閉</string>
```

Comment:
You should translate this on transifex when the source file is updated, because it will probably will be overwritten.